### PR TITLE
Add Performance Setup to Project Creation

### PR DESCRIPTION
**Pull request by silent1mezzo**
This adds documentation to setup sending transactions for Javascript, React, Python and Django projects.

### CODE REVIEW

**Comment by silent1mezzo** [comment]
I'm not 100% confident in https://github.com/getsentry/sentry-docs/pull/2031/files#diff-659d61edb291833eaf510b9baf390599R12. I haven't been able to spin this up locally yet to test it out.

File: src/wizard/javascript/index.md [hunk 1]
```diff
@@ -4,28 +4,33 @@ doc_link: https://docs.sentry.io/error-reporting/quickstart/?platform=browser
...
 type: language
 ---
+
 The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
 
 ```html
 <script
-  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
-  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.apm.min.js"
```
Comment (dcramer) [inline src/wizard/javascript/index.md:12]: might be worth a small Note: callout that says they can view the full docs if they want to narrow the scope of the bundle

Comment (silent1mezzo) [reply src/wizard/javascript/index.md:12]: @dcramer I searched the docs but couldn't find a list of bundles avail. Does that exist?

Comment (silent1mezzo) [reply src/wizard/javascript/index.md:12]: Going to launch this and then update with a note once I know where to point it to.

Comment (dcramer) [reply src/wizard/javascript/index.md:12]: probably not cc @HazAT

**Pull request merged**